.gitignore

```diff
@@ -33,6 +33,7 @@ devtools/build
 server/log.txt
 pack-stats.json
 default.profraw
+concrete-storage/package-lock.json
```

Comment:
Delete this? (looks like you had a package-lock.json file earlier in this PR, but since removed it?)

Reply:
Walt suggested I disable creation of `package-lock.json` via an `.npmrc`. I chose a middle ground which is to just eliminate it from commits. 

IOW, it's removed from this PR _because_ it's in .gitignore.